scraper/src/kolibri2zim/scraper.py

```diff
@@ -811,7 +967,7 @@ def run(self):
             return 1
         self.creator = Creator(
             filename=self.output_dir.joinpath(self.clean_fname),
-            main_path=self.root_id,
+            main_path="index.html",  # self.root_id,
```

Comment:
The comment is ugly yes.
`index.html` is ... usual?

Reply:
No it's not. Web servers tend to alias `/` with an index file (`index.html` and others) so while there is an `index.html` file, the URL doesn't show it.
But there is no web server here, that's a ZIM and the URL will be visible

Reply:
What would you prefer? Having a dynamic name based on channel title is going to be quite complex.

Reply:
I think `""` (empty path) is best but we've also used `"home"` somwehere.

Reply:
I failed to make it work with `""`.

Kiwix-server is not happy with it. Looks like it is not loading the Vue.JS main page with a trailing "/" in the URL, and so Vue.JS is not loading its subresources properly.

Kiwix JS complains that "The main page of this archive does not seem to be an article".

Both are working well with same code base but just `"home"` instead of `""` as main path.

Reply:
kiwix-serve works fine with an empty path as mainPage but indeed the in-zim tools might have unmet expectations. `home` is fine 👍